app/grandchallenge/components/models.py

```diff
@@ -268,6 +268,7 @@ def interface_type_annotation():
                     ],
                     "sub_type": "brush",
                     "groups": []
+                }
```

Comment:
Nitpicky, but should we add extra indentation for the objects in the list? Having the `}` and `]` with the same indentation level threw me off a little.

Reply:
Easy fix, it did take me a while to find what was wrong with the JSON.